kong/llm/drivers/shared.lua

```diff
@@ -56,15 +61,25 @@ _M.streaming_has_token_counts = {
   ["llama2"] = true,
   ["anthropic"] = true,
   ["gemini"] = true,
+  ["bedrock"] = true,
+}
+
+_M.bedrock_unsupported_system_role_patterns = {
```

Comment:
should this be under the `bedrock.lua` file instead?

Reply:
Yes, whoops